pom.xml

```diff
@@ -286,6 +286,31 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
```

Comment:
for some reason this does not work when I add it to the plugin-management section